go.mod

```diff
@@ -9,29 +9,29 @@ require (
 	github.com/cpuguy83/go-md2man v1.0.10
 	github.com/fatih/color v1.9.0
 	github.com/ghodss/yaml v1.0.0
-	github.com/google/go-cmp v0.5.0
+	github.com/google/go-cmp v0.5.2
 	github.com/hako/durafmt v0.0.0-20191009132224-3f39dc1ed9f4
 	github.com/hinshun/vt10x v0.0.0-20180809195222-d55458df857c
 	github.com/jonboulle/clockwork v0.1.1-0.20190114141812-62fb9bc030d1
 	github.com/ktr0731/go-fuzzyfinder v0.2.0
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
-	github.com/tektoncd/pipeline v0.16.3
+	github.com/tektoncd/pipeline v0.17.1-0.20201006183654-1710b688c023
```

Comment:
is it fine to bump to a commit in master branch, we may need first to bump to 0.17 branch and then do the release based on 0.17.x

Reply:
I think it's fine, we don't necessary need to do a release for each pipeline release. but, to be discussed